### ISSUE TITLE
Problems installing plugin

### ISSUE THREAD

**Issue by maollano** [pluginsGLPI/formcreator#816]
Dear
I am trying to install the Plugin on version 9.2, and it shows the following error:

**_Database Design Change - glpi_plugin_formcreator_answers (0 seconds)

2.6.0 multiple alter in glpi_plugin_formcreator_answers - Error during query to database: ALTER TABLE `glpi_plugin_formcreator_answers` ADD INDEX` plugin_formcreator_question_id` (`plugin_formcreator_question_id`) - Error is Key column 'plugin_formcreator_question_id' does not exist in table_**

follow the instructions in thread #794 but it does not work, did not have previous installations of this plugin.

Best regards

Mauro

**Comment by btry** [comment]
Hi

If you never installed the plugin and have this error then something makes the installer wrong and believe it shall upgrade instead of install.

I did a try. I succeeded to make the plugin fail (weird success). Theck in the database. I expect you have a table whose the name begins with **glpi_plugin_formcreator_**

Having at least one table with such name will lead the installer to take  the wrong decision. And the migration class will fail with a wrong message.
